### PR TITLE
[pt-br] Add docs/tasks/configure-pod-container/configure-volume-storage.md

### PR DESCRIPTION
**Pull request by DonatoHorn**

### CODE REVIEW

File: content/pt-br/docs/tasks/configure-pod-container/configure-volume-storage.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+    redis     1/1       Running    0          13s
+    redis     0/1       Completed  0         6m
+    redis     1/1       Running    1         6m
+    ```
+
+Neste ponto, o Contêiner terminou e reiniciou. Isso porque o Pod do Redis tem uma
+[`restartPolicy`](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#podspec-v1-core)
+de `Always`.
+
+1. Abra um shell dentro do Contêiner reiniciado:
```
Comment (sftim) [inline content/pt-br/docs/tasks/configure-pod-container/configure-volume-storage.md:113]: If writing the translated term “Contêiner”, don't capitalize the first letter.

Comment (DonatoHorn) [reply content/pt-br/docs/tasks/configure-pod-container/configure-volume-storage.md:113]: Just following the origin en docs page.

Comment (sftim) [reply content/pt-br/docs/tasks/configure-pod-container/configure-volume-storage.md:113]: In English, “Container” is correct; the term in the API is `Container` and the local word is “container”.

Portugues uses the word “contêiner” but the API field is still `container`, whatever language the reader speaks.

**Review by stormqueen1990 — COMMENTED** [review]
Hello there, @DonatoHorn! 👋

Thank you very much for your contribution! 🎉 

/lgtm

**Comment by stormqueen1990** [comment]
/retitle [pt-br] Add docs/tasks/configure-pod-container/configure-volume-storage.md

**Comment by stormqueen1990** [comment]
/cc @devlware @edsoncelio 
Could you please provide a second review?

**Pull request merged**